exporter/sumologicexporter/documentation.md

```diff
@@ -8,32 +8,32 @@ The following telemetry is emitted by this component.
 
 ### otelcol_exporter_requests_bytes
 
-Total size of requests (in bytes)
+Total size of requests (in bytes) [development]
 
-| Unit | Metric Type | Value Type | Monotonic |
-| ---- | ----------- | ---------- | --------- |
-| By | Sum | Int | true |
+| Unit | Metric Type | Value Type | Monotonic | Stability |
+| ---- | ----------- | ---------- | --------- | --------- |
+| By | Sum | Int | true | development |
```

Comment:
shouldn't this be `Beta`

Reply:
Metrics can be in "lower" stability level from what the component defines. Even for a stable component, we can have new metrics that are still in `development`. 

https://github.com/open-telemetry/opentelemetry-collector/issues/13910 will define the ruling around this, but this PR starts with setting the baseline.